xml/planning-planning-compute-node_input_model.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
```

Comment:
If in the end, you do need this new file (see my first comment), please give it a shorter name, e.g. `planning-compute-model.xml` or so (use `git mv OLD_NAME NEW_NAME` to do that). The `planning-planning` stuff comes from dietrich and we somewhat missed the chance to rename the initial files when it was still painless.